### ISSUE TITLE
Accessibility Issue: Links must have discernible text  /buy-vs-build-start.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#737]
**Description**
link must have discernible text, cannot be just an image  

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/buy-vs-build-start.html
*CSS Selector(s) (if possible and applicable)*:  p:nth-child(4) > a
*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

<img width="538" alt="Screen Shot 2020-01-31 at 10 32 27 AM" src="https://user-images.githubusercontent.com/26819992/73551759-01989400-4415-11ea-868d-9ac285b5a000.png">

**Additional information or thoughts**

As a result of #699
